operations/host_spawn.go

```diff
@@ -480,6 +480,58 @@ func hostStart() cli.Command {
 	}
 }
 
+func hostSSH() cli.Command {
+	const (
+		keyFlagName = "key"
+	)
+
+	return cli.Command{
+		Name:  "ssh",
+		Usage: "ssh into a spawn host",
+		Flags: addHostFlag(
+			cli.StringFlag{
+				Name:  joinFlagNames(keyFlagName, "k"),
+				Usage: "name of a public key to use",
+			},
+		),
+		Before: mergeBeforeFuncs(setPlainLogger, requireHostFlag),
+		Action: func(c *cli.Context) error {
+			confPath := c.Parent().Parent().String(confFlagName)
+			hostID := c.String(hostFlagName)
+			key := c.String(keyFlagName)
+
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			conf, err := NewClientSettings(confPath)
+			if err != nil {
+				return errors.Wrap(err, "problem loading configuration")
+			}
+			client := conf.setupRestCommunicator(ctx)
+			defer client.Close()
+
+			h, err := client.GetSpawnHost(ctx, hostID)
+			if err != nil {
+				return errors.Wrap(err, "problem getting host")
+			}
+			if h == nil {
+				return errors.New("host not found")
+			}
+			if h.Status == nil || *h.Status != evergreen.HostRunning {
+				return errors.New("host is not running")
+			}
+			if h.User == nil || h.HostURL == nil {
+				return errors.New("unable to ssh into host without user or DNS name")
+			}
+			args := []string{"ssh", "-tt", fmt.Sprintf("%s@%s", *h.User, *h.HostURL)}
```

Comment:
Ah I didn't realize that. Do you know how to verify if ssh is on the path? If it's not on the path then it will return an error to the user right? I would expect users to know that ssh should be on the path by this point.

Reply:
I think we can assume people have ssh on their PATH. Certainly the macbooks we provide and any standard Linux distro someone might be using would have it. Also, if they don't have it, I think the error would be clear.